app/backend/app.py

```diff
@@ -36,24 +36,20 @@
 from approaches.chatreadretrievereadvision import ChatReadRetrieveReadVisionApproach
 from approaches.retrievethenread import RetrieveThenReadApproach
 from approaches.retrievethenreadvision import RetrieveThenReadVisionApproach
+from config import (
```

Comment:
I think we didn't end up saving lines of code with moving them to a file, but I don't have a strong preference either way.